src/tests/map-constructor.ts

```diff
@@ -59,3 +59,16 @@ doNotExecute(() => {
 
   type testClear = [Expect<Equal<typeof cleared, void>>];
 });
+
+doNotExecute(() => {
+  const map = new Map() satisfies Map<string, boolean>;
+  type test = [Expect<Equal<typeof map, Map<string, boolean>>>];
+});
+
+doNotExecute(() => {
```

Comment:
Could you add another test, testing:

`const map: Map<string, boolean> = new Map();`

Reply:
Of course.

Want me to also add a test with passing to a function accepting a specific map?
Or is that included in one of the 2?

Reply:
Why not!

Reply:
Done & done!